tools/api-markdown-documenter/src/test/test-data/README.md

```diff
@@ -0,0 +1,3 @@
+Each directory under this folder represents an "API Model".
```

Comment:
Is "API Model" defined somewhere in the documentation?

Reply:
Yeah, it's defined in the package README (by linking to external documentation).

Reply:
Can you link it here?

Reply:
Our docs: https://github.com/microsoft/FluidFramework/tree/main/tools/api-markdown-documenter#transformapimodel

External: https://api.rushstack.io/pages/api-extractor-model.apimodel/

That said, it doesn't look like the docs do a great job of explaining how a directory of API report files corresponds to a model. The package README could probably use a glossary of terms at the top. I'll file an item to do that.